R/zzz.R

```diff
@@ -34,3 +34,5 @@ RcodeBlock <- getFromNamespace("RcodeBlock", "teal.reporter") # nolint: object_n
 # Use non-exported function(s) from teal.code
 # This one is here because lang2calls should not be exported from teal.code
 lang2calls <- getFromNamespace("lang2calls", "teal.code")
+
+get_code_dependency <- getFromNamespace("get_code_dependency", "teal.data")
```

Comment:
Just double checking it is fine to pull, since it's not exported and internal https://github.com/insightsengineering/teal.data/blob/main/R/utils-get_code_dependency.R#L27

Reply:
It is okey, the same we do with other functions, like `long2calls` (two lines above)

Reply:
This is a work-around to lure cran checks by not using `:::`